app/views/application/_messages.html.erb

```diff
@@ -1,3 +1,18 @@
+<% if flash.key?(:alert) %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+        <h2 class="govuk-error-summary__title" id="error-summary-title">
+          There is a problem
+        </h2>
+        <div class="govuk-error-summary__body">
+          <p class="govuk-body"><%= flash[:alert] %></p>
+        </div>
+      </div>
+    </div>
+  </div>
+<% end %>
```

Comment:
Not sure this is the right component here - should probably use a notification banner:

https://github.com/alphagov/govuk-design-system-backlog/issues/46#issuecomment-576316811